docs/extras/expression_language/how_to/routing.ipynb

```diff
@@ -92,44 +112,147 @@
     "Respond to the following question:\n",
     "\n",
     "Question: {question}\n",
-    "Answer:\"\"\") | ChatOpenAI()\n",
-    "openai_chain = PromptTemplate.from_template(\"\"\"You are an expert in openai. \\\n",
-    "Always answer questions starting with \"As Sam Altman told me\". \\\n",
+    "Answer:\"\"\") | ChatAnthropic()\n",
+    "anthropic_chain = PromptTemplate.from_template(\"\"\"You are an expert in anthropic. \\\n",
+    "Always answer questions starting with \"As Dario Amodei told me\". \\\n",
     "Respond to the following question:\n",
     "\n",
     "Question: {question}\n",
-    "Answer:\"\"\") | ChatOpenAI()\n",
+    "Answer:\"\"\") | ChatAnthropic()\n",
     "general_chain = PromptTemplate.from_template(\"\"\"Respond to the following question:\n",
     "\n",
     "Question: {question}\n",
-    "Answer:\"\"\") | ChatOpenAI()"
+    "Answer:\"\"\") | ChatAnthropic()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "593eab06",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain.schema.runnable import RunnableBranch\n",
+    "\n",
+    "branch = RunnableBranch(\n",
+    "  (lambda x: \"anthropic\" in x[\"topic\"].lower(), (lambda x: {\"question\": x[\"question\"]}) | anthropic_chain),\n",
```

Comment:
Would be nice to just have e.g. `anthropic_chain` directly as the second argument of the tuple, but it throws an error :(